cryptol-remote-api/cryptol-remote-api.cabal

```diff
@@ -40,19 +40,19 @@ common errors
 
 common deps
   build-depends:
-    base                 >=4.11.1.0 && <4.19,
+    base                 >=4.11.1.0 && <4.20,
     argo,
     aeson                >= 1.4.2 && < 2.3,
     base64-bytestring    >= 1.0,
-    bytestring           >= 0.10.8 && < 0.12,
+    bytestring           >= 0.10.8 && < 0.13,
```

Comment:
I wonder if we could just stop specifying upper (or any) bounds on bytestring and text (and possibly any others listed at https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history) because since they are distributed as boot libraries along with GHC, they are effectively constrained by the `base` bounds?  This would mean fewer places needing an update when we do these version bumps.

Reply:
These boot library versions aren't wholly determined by `base`, as it's still possible to manually specify different versions of boot libraries than the ones that come included with GHC. (Indeed, some of our users do exactly this, as seen [here](https://github.com/GaloisInc/what4/pull/254).) Personally, I'd rather err on the side of caution here, since it is quite possible for major version bumps in boot libraries to break things (see, for instance, the `mtl-2.3.*`-related changes in #1572).